testdata/server-discovery-and-monitoring/rs/too_old.yml

```diff
@@ -18,7 +18,9 @@ phases: [
                     isWritablePrimary: false,
                     secondary: true,
                     setName: "rs",
-                    hosts: ["a:27017", "b:27017"]
+                    hosts: ["a:27017", "b:27017"],
```

Comment:
The YAML files are the human-readable part of the integration test. The `TestSDAMSpec` test [uses the `.json` analogues ](https://github.com/mongodb/mongo-go-driver/blob/44684f4965da712e0d15c7611c82a46d8a4bd66b/x/mongo/driver/topology/sdam_spec_test.go#L563) to decode the test into the driver-specific [testCase](https://github.com/mongodb/mongo-go-driver/blob/v1/x/mongo/driver/topology/sdam_spec_test.go#L170). The changes required to sync are defined at https://github.com/mongodb/specifications/commit/a0845453aa68fc10be0242dc5ca5f29f9dd526d8 

Two things to note:

1. Drivers generally can chose to test YAML or JSON, but the Go Driver uses JSON. However, we always want to include both files. You can read more about that in the [unified test format](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.md#unified-test-format) documentation.
2. The SDAM tests are otherwise not bounded to the unified test format, they have a more specific design defined here: https://github.com/mongodb/specifications/tree/master/source/server-discovery-and-monitoring/tests#format

Reply:
Got it! Made the requested changes